continuous-deployment/readme.md

```diff
@@ -1,7 +1,142 @@
-# Continuous Deployment
+# Continuous Delivery
```

Comment:
We should probably update the folder name too but, maybe more appropriate for follow-up PR. That way we dont see everything showing as changed.